style.css

```diff
@@ -1,11 +1,82 @@
 
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
 input {
   display: block;
 }
 
-#form {
+h1,
+h2 {
+  text-align: center;
+}
+
+.books {
+  background-color: lavender;
   display: flex;
   flex-direction: column;
+  border: 2px solid black;
+  border-radius: 0.4em;
+  margin: 2rem;
+}
+
+.book {
+  display: flex;
   align-items: center;
+  justify-content: space-between;
+  padding: 0.4em 0;
+}
+
+.book:nth-child(odd) {
+  background-color: rgba(0, 0, 0, 0.1);
+}
+
+.book button {
+  background-color: whitesmoke;
+  border: 2px solid gray;
+  padding: 0.5em 1em;
+  border-radius: 0.2em;
+  font-weight: 600;
+  margin: 0 1em;
+  cursor: pointer;
+}
+
+.book p {
+  margin: 0 1em;
+}
+
+.bookInput {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5em;
   justify-content: center;
+  align-items: flex-end;
+  width: 20rem;
+  margin: 4rem auto;
+}
```

Comment:
- [ ] <b>REQUIRED:</b> As the project requires, you can use styles of your choice (colors, fonts, backgrounds, etc.) but the general layout should be followed as much as possible. with that being said, the `add new book` form should be centered and it should have the title `Add a new book` as is in the provided mockup. kindly fix it.

| Mockup      | Yours |
  | ----------- | ----------- |
| ![80047697-0c3f6280-8506-11ea-9416-a4f1800e2329](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png?raw=true) | ![scrnli_9_23_2021_2-44-05 AM](https://user-images.githubusercontent.com/45217364/134440360-7ca48e3a-cf0c-465a-98f4-00878f8ac0b1.png) |